src/app/Dashboard/Quickstart/quickstarts/add-card-quickstart.ts

```diff
@@ -0,0 +1,83 @@
+ * Works (as defined below), to deal in both
+ *
+ * (a) the Software, and
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import { QuickStart } from '@patternfly/quickstarts';
+import { cryostatQuickStartIcon } from './cryostat-icon';
+export const addCardQuickStart: QuickStart = {
+  metadata: {
+    name: 'add-card-quickstart',
+  },
+  spec: {
+    displayName: 'Adding/Removing Cards',
+    durationMinutes: 1,
+    icon: cryostatQuickStartIcon,
+    description: 'Add and remove cards from the Cryostat Dashboard.',
+    introduction: '### This is a sample quickstart.',
+    tasks: [
+      {
+        title: 'Add a Card',
+        description: `### We will add a card to the Cryostat Dashboard.
+1. Go to the bottom of this page, and press the **Add** button on the 'Add Card' card.
+2. Select a card type.
+3. Press **Finish** to add the card to the dashboard.`,
+        review: {
+          instructions: `#### To verify the card was added:
+1. Notice the new card located on the Dashboard.
+                    `,
+          failedTaskHelp: 'Try the steps again.',
+        },
+        summary: {
+          success: 'You have successfully added a card.',
+          failed: 'Try the steps again.',
+        },
+      },
+      {
+        title: 'Remove a Card',
+        description: `### We will remove a card to the Cryostat Dashboard.
+1. Find the card was created in Step 1.
+2. Press the kebab on the top right of that card's header.
+3. Press **Remove Card**.`,
+        summary: {
+          success: 'You have succesfully removed a card.',
+          failed: 'Try the steps again.',
+        },
+      },
+    ],
+    prerequisites: ['sample-quickstart'],
```

Comment:
Looks like this one is an instruction text (no requirement to be an id). So maybe we can have it as

```ts
prerequisites: ['Complete the **sample-quickstart** quick start.'],
```

![Screenshot from 2023-02-16 22-11-57](https://user-images.githubusercontent.com/68053619/219540190-f5fd629f-fc5a-4816-861a-6a874eb48d2c.png)